core/data_migration_scripts/4_1_0_stages/EE_DMS_4_1_0_attendees.dmsstage.php

```diff
@@ -5,85 +5,55 @@
-  `event_type` varchar(250) DEFAULT NULL,
-  `allow_overflow` varchar(1) DEFAULT 'N',
-  `overflow_event_id` int(10) DEFAULT '0',
-  `recurrence_id` int(11) DEFAULT '0',
-  `email_id` int(11) DEFAULT '0',
-  `alt_email` text,
-  `event_meta` longtext,
-  `wp_user` int(22) DEFAULT '1',
-  `require_pre_approval` int(11) DEFAULT '0',
-  `timezone_string` varchar(250) DEFAULT NULL,
-  `likes` int(22) DEFAULT NULL,
-  `ticket_id` int(22) DEFAULT '0',
-  `submitted` datetime NOT NULL,
-  PRIMARY KEY (`id`),
-  KEY `event_code` (`event_code`),
-  KEY `wp_user` (`wp_user`),
-  KEY `event_name` (`event_name`),
-  KEY `city` (`city`),
-  KEY `state` (`state`),
-  KEY `start_date` (`start_date`),
-  KEY `end_date` (`end_date`),
-  KEY `registration_start` (`registration_start`),
-  KEY `registration_end` (`registration_end`),
-  KEY `reg_limit` (`reg_limit`),
-  KEY `event_status` (`event_status`),
-  KEY `recurrence_id` (`recurrence_id`),
-  KEY `submitted` (`submitted`),
-  KEY `likes` (`likes`)
-) ENGINE=InnoDB DEFAULT CHARSET=utf8$$
+CREATE TABLE `wp_events_attendee` (
+`id` INT(11) UNSIGNED NOT NULL AUTO_INCREMENT,
+`registration_id` VARCHAR(23) NULL DEFAULT '0',
+`is_primary` TINYINT(1) NULL DEFAULT '0',
+`lname` VARCHAR(45) NULL DEFAULT NULL,
+`fname` VARCHAR(45) NULL DEFAULT NULL,
+`address` VARCHAR(255) NULL DEFAULT NULL,
+`address2` VARCHAR(255) NULL DEFAULT NULL,
+`city` VARCHAR(45) NULL DEFAULT NULL,
+`state` VARCHAR(45) NULL DEFAULT NULL,
+`zip` VARCHAR(45) NULL DEFAULT NULL,
+`country_id` VARCHAR(128) NULL DEFAULT NULL,
+`organization_name` VARCHAR(50) NULL DEFAULT NULL,
+`vat_number` VARCHAR(20) NULL DEFAULT NULL,
+`email` VARCHAR(255) NULL DEFAULT NULL,
+`phone` VARCHAR(45) NULL DEFAULT NULL,
+`date` TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+`price_option` VARCHAR(150) NULL DEFAULT NULL,
+`orig_price` DECIMAL(20,2) NULL DEFAULT '0.00',
+`final_price` DECIMAL(20,2) NULL DEFAULT '0.00',
+`quantity` INT(11) NULL DEFAULT '0',
+`total_cost` DECIMAL(20,2) NULL DEFAULT '0.00',
+`amount_pd` DECIMAL(20,2) NULL DEFAULT '0.00',
+`coupon_code` VARCHAR(45) NULL DEFAULT NULL,
+`payment` VARCHAR(45) NULL DEFAULT NULL,
+`payment_status` VARCHAR(45) NULL DEFAULT 'Incomplete',
+`txn_type` VARCHAR(45) NULL DEFAULT NULL,
+`txn_id` VARCHAR(250) NULL DEFAULT NULL,
+`payment_date` VARCHAR(45) NULL DEFAULT NULL,
+`event_id` VARCHAR(45) NULL DEFAULT NULL,
+`event_time` VARCHAR(15) NULL DEFAULT NULL,
+`end_time` VARCHAR(15) NULL DEFAULT NULL,
+`start_date` VARCHAR(45) NULL DEFAULT NULL,
+`end_date` VARCHAR(45) NULL DEFAULT NULL,
+`attendee_session` VARCHAR(250) NULL DEFAULT NULL,
+`transaction_details` TEXT NULL,
+`pre_approve` INT(11) NULL DEFAULT '1',
+`checked_in` INT(1) NULL DEFAULT '0',
+`checked_in_quantity` INT(11) NULL DEFAULT '0',
+`hashSalt` VARCHAR(250) NULL DEFAULT NULL,
+PRIMARY KEY (`id`),
+INDEX `registration_id` (`registration_id`),
+INDEX `event_id` (`event_id`)
+)
+COLLATE='utf8_general_ci'
+ENGINE=InnoDB
+AUTO_INCREMENT=3
+;
+
```

Comment:
This seems like an odd change.  Why is the creation of `wp_events_detail` table removed, and the creation of `wp_events_attendee` added?

Reply:
Ah, ya this does look pretty weird. What you might not see from the summary is this is actually just a comment. When originally writing these DMSs, I put the relevant EE3 table definitions in comments for reference... but on this class which migrated attendees I had previously put the wrong EE3 table definition in the comment- the events table. So here I'm instead showing the old attendees table, as that's what this class migrates from. Does that make more sense?

Reply:
Oh I see.  It's also breaking standards a bit because every line of a phpdoc block should have a `*` line delimiter (which the comments here don't have).  I'm not going to be picky on it though if you don't want to fix.